stumpy/scraamp.py

```diff
@@ -45,6 +102,12 @@ def _compute_PI(
             zone_start = max(0, i - excl_zone)
             zone_stop = min(l, i + excl_zone)
             p_norm_profile[zone_start : zone_stop + 1] = np.inf
+
+            # only for self-join
+            mask = p_norm_profile < P_NORM[thread_idx]
+            P_NORM[thread_idx][mask] = p_norm_profile[mask]
```

Comment:
Can this be `P_NORM[thread_idx, mask]`? And same with the next line>

Reply:
Sure. I will revise both.